### PR TITLE
User-guide: Add ironic dnsmasq DNS & GW IP environment

### PR DESCRIPTION
**Pull request by shibaPuppy**
- https://github.com/metal3-io/ironic-image/pull/389
- description of the PR above.

### CODE REVIEW

**Review by elfosardo — CHANGES_REQUESTED** [review]

File: docs/user-guide/src/ironic/ironic_variables.md [hunk 1]
```diff
@@ -18,5 +18,7 @@ The following environmental variables can be passed to configure the Ironic serv
 - `IPA_DOWNLOAD_ENABLED` - enables the use of the Ironic Python Agent Downloader container to download IPA archive (default true)
 - `USE_LOCAL_IPA` - enables the use of locally supplied IPA archive. This condition is handled by BMO and this has effect only when `IPA_DOWNLOAD_ENABLED` is "false", otherwise `IPA_DOWNLOAD_ENABLED` takes precedence. (default false)
 - `LOCAL_IPA_PATH` - this has effect only when `USE_LOCAL_IPA` is set to "true", points to the directory where the IPA archive is located. This variable is handled by BMO. The variable should contain an arbitrary path pointing to the directory that contains the ironic-python-agent.tar
+- `GW_IP` - gateway IP address to use for ironic dnsmasq(dhcpd)
```
[the remark this answers is not in the document]
Comment (shibaPuppy) [reply docs/user-guide/src/ironic/ironic_variables.md]: @elfosardo 
https://github.com/metal3-io/ironic-image/pull/389#discussion_r1001791766
change from `GW_IP` to `GATEWAY_IP` due to the above comment

[the remark this answers is not in the document]
Comment (elfosardo) [reply docs/user-guide/src/ironic/ironic_variables.md]: thanks, can you please also expand the definitions as suggested?

[the remark this answers is not in the document]
Comment (shibaPuppy) [reply docs/user-guide/src/ironic/ironic_variables.md]: @elfosardo 
yes, it has been merged, documentation need to be updated.
https://github.com/metal3-io/ironic-image/pull/389

File: docs/user-guide/src/ironic/ironic_variables.md [hunk 2]
```diff
@@ -18,5 +18,7 @@ The following environmental variables can be passed to configure the Ironic serv
 - `IPA_DOWNLOAD_ENABLED` - enables the use of the Ironic Python Agent Downloader container to download IPA archive (default true)
 - `USE_LOCAL_IPA` - enables the use of locally supplied IPA archive. This condition is handled by BMO and this has effect only when `IPA_DOWNLOAD_ENABLED` is "false", otherwise `IPA_DOWNLOAD_ENABLED` takes precedence. (default false)
 - `LOCAL_IPA_PATH` - this has effect only when `USE_LOCAL_IPA` is set to "true", points to the directory where the IPA archive is located. This variable is handled by BMO. The variable should contain an arbitrary path pointing to the directory that contains the ironic-python-agent.tar
+- `GATEWAY_IP` - gateway IP address to use for ironic dnsmasq(dhcpd)
+- `DNS_IP` - DNS IP address to use for ironic dnsmasq(dhcpd)
```
Comment (furkatgofurov7) [inline docs/user-guide/src/ironic/ironic_variables.md:22]:
```suggestion
- `GATEWAY_IP` - gateway IP address to use for ironic dnsmasq (dhcpd)
- `DNS_IP` - DNS IP address to use for ironic dnsmasq (dhcpd)
```

Comment (shibaPuppy) [reply docs/user-guide/src/ironic/ironic_variables.md]: @furkatgofurov7 
ok, it has been corrected.

I didn't want to make a new commit, so forced push.

Comment (shibaPuppy) [reply docs/user-guide/src/ironic/ironic_variables.md]: retest plz

**Review by furkatgofurov7 — COMMENTED** [review]
Forgot this completely, thanks for the PR @shibaPuppy 

/lgtm

**Pull request merged**